src/main/java/com/hermesworld/ais/galapagos/topics/MessagesSize.java

```diff
@@ -1,6 +1,15 @@
 package com.hermesworld.ais.galapagos.topics;
 
+import lombok.Getter;
+
+@Getter
 public enum MessagesSize {
 
-    VERY_SMALL, SMALL, NORMAL, LARGE, VERY_LARGE;
+    VERY_SMALL(1000), SMALL(10000), NORMAL(100000), LARGE(125000), VERY_LARGE(150000);
```

Comment:
Same as above, please check that this matches values from ticket (I don't see 125000 or 150000 there anywhere).